epub33/rs/index.html

```diff
@@ -173,11 +173,10 @@ <h2>EPUB Publications</h2>
 								[[!EPUB-33]].</p>
 						</li>
 						<li>
-							<p id="confreq-rs-data-urls">It MUST prevent attemps at top-level navigation to data URLs
-								[[!RFC2397]]. (Examples of top-level navigation include loading resources not in the
-								spine as though they are <a>Top-Level Content Documents</a> and opening <a
+							<p id="confreq-rs-data-urls">It MUST prevent data URLs [[!RFC2397]] from opening in <a
 									href="https://html.spec.whatwg.org/multipage/browsers.html#top-level-browsing-context"
-									>top-level browsing contexts</a> [[HTML]])</p>
+									>top-level browsing contexts</a> [[HTML]], except when initiated through the user
+									interface of the Reading System (e.g., a context menu).</p>
```

Comment:
Yeah, this language is tricksy :( Isn't clicking on something to make something happen part of the user interface of the RS? Not sure how to best phrase this.

Reply:
> Isn't clicking on something to make something happen part of the user interface

Ya, it took a few tries to even come up with this. I wouldn't consider clicking in the content the same as the clicking a part of the user interface, but it is awkward.

Would "affordance" improve it any: "except when initiated through a Reading System affordance such as a context menu"?

Reply:
I've updated this in the latest commit.